test/e2e/support/index.ts

```diff
@@ -49,3 +49,11 @@ afterEach(function () {
     });
   }
 });
+
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/;
```

Comment:
CodeQL complains a lot here, and the regex looks a bit odd. Maybe this could work instead?

```suggestion
const resizeObserverLoopErrRe = /^ResizeObserver loop limit exceeded/;
```

And, we should probably have a proper description in a comment here on why we need to ignore the error (it's not just for forcing the test to pass). It should probably also be noted that it should be removed if possible, when the design system (hopefully) fixes the root cause.

Reply:
Otherwise, if the `[` and `]` were supposed to be literal, they should be escaped in the regex.

Reply:
this is straight up copied, and I don't know why everyone uses a regex like that

Reply:
Turns out someone else pointed that out too https://github.com/cypress-io/cypress/issues/8418#issuecomment-992564877

Reply:
Ah, nice! That's an even better solution.